classes/Project/Project.mjs

```diff
@@ -45,25 +48,19 @@ export default class Project {
     return database.remove(projectId)
   }
 
-  async getProject(projectId) {
+  async getById(projectId) {
     if (!projectId) {
       err_out.message = "Project ID is required"
       err_out.status = 400
       throw err_out
     }
     return database
-      .getById(projectId)
-      .then((resp) => {
-        resp.json()
-      })
-      .catch((error) => {
-        console.log(error)
-        throw error
-      })
+      .getById(projectId, "Project")
+      .then((resp) => (resp?.length ? resp[0] : resp))
```

Comment:
What are we expecting? I think if `getById` gives us more than just one thing, we are doing it wrong.

Reply:
`getById` uses databse find() which returns an array. in this case, it'd always be an array of length 1. hence resp[0]. I put resp as a fail safe in case we rewrite the getById to return an object